src/Surfnet/Stepup/Identity/Event/EmailVerifiedEvent.php

```diff
@@ -79,20 +74,18 @@ public function __construct(
         SecondFactorId $secondFactorId,
         SecondFactorType $secondFactorType,
         DateTime $registrationRequestedAt,
+        IdentifyingDataId $identifyingDataId,
         $registrationCode,
```

Comment:
Choose to go with "typehint-able" first, "non-typehintable" later. Adding chores for moving everything to Value objects is on my todo-list :wink: